rts/Lua/LuaTextures.cpp

```diff
@@ -29,6 +29,111 @@ const spring::unordered_map<GLenum, GLenum> LuaTextures::Format2Query =
 /******************************************************************************/
 /******************************************************************************/
 
+namespace Impl {
+	static GLenum GetInternalFormatDataFormat(GLenum internalFormat) {
+		GLenum dataFormat;
+		switch (internalFormat) {
+			case GL_R8UI:
+			case GL_R16UI:
+			case GL_R32UI: {
+				dataFormat = GL_RED_INTEGER;
```

Comment:
is the variable needed? how about
```suggestion
				return GL_RED_INTEGER;
```

Reply:
> is the variable needed? how about

Will that be preferable?
I think it makes sense to use a variable for better readability, to see what it is we are returning, because the list may grow 1 screen more, so we are not mistaking one function by another when looking at the code.